app.config.js

```diff
@@ -1,44 +1,74 @@
+  API_KEY: 'AIzaSyBFiuUulmuQ7Pv1VvxpUQB01AWCEQhIToA',
+  AUTH_DOMAIN: 'npm-otp-f6bfc.firebaseapp.com',
+  DATABASE_URL: 'https://npm-otp-f6bfc.firebaseio.com',
+  PROJECT_ID: 'npm-otp-f6bfc',
+  STORAGE_BUCKET: 'npm-otp-f6bfc.appspot.com',
+  MESSAGING_SENDER_ID: '230076165693',
+  APP_ID: '1:230076165693:web:a04f9ad6f64be8ec248454',
+  CLIENT_ID:
+    '230076165693-0mj3vb13158tnru89f1re89m9o94g8e7.apps.googleusercontent.com',
+  CLIENT_ID_ANDROID:
+    '230076165693-a668m77v43bsbi0usea3p4o83ehco09r.apps.googleusercontent.com',
+  CLIENT_ID_IOS:
+    '230076165693-v8kk0ase89jp3s1qe0rtg70t3vb2h9i9.apps.googleusercontent.com',
+  EAS_PROJECT_ID: 'e3c6e759-c8c5-41a8-861e-b3e4adab619a',
+  APP_PACKAGE_ID: 'com.nearform.optic',
+}
+
 export default {
-  expo: {
-    name: 'optic-expo',
-    slug: 'optic-expo',
-    owner: 'nearform',
-    scheme: 'optic-expo',
-    entryPoint: './src/index.js',
-    version: '1.0.0',
-    orientation: 'portrait',
-    icon: './src/assets/icon.png',
-    extra: {
-      apiUrl: process.env.API_URL,
-      apiKey: process.env.API_KEY,
-      authDomain: process.env.AUTH_DOMAIN,
-      databaseURL: process.env.DATABASE_URL,
-      projectId: process.env.PROJECT_ID,
-      storageBucket: process.env.STORAGE_BUCKET,
-      messagingSenderId: process.env.MESSAGING_SENDER_ID,
-      appId: process.env.APP_ID,
-      clientId: process.env.CLIENT_ID,
-    },
-    splash: {
-      image: './src/assets/splash.png',
-      resizeMode: 'contain',
-      backgroundColor: '#ffffff',
-    },
-    updates: {
-      fallbackToCacheTimeout: 0,
-    },
-    assetBundlePatterns: ['**/*'],
-    platforms: ['android', 'ios'],
-    ios: {
-      supportsTablet: true,
+  name: 'optic',
+  slug: 'optic-expo',
+  owner: 'nearform',
+  scheme: 'optic-expo',
+  version: '1.0.4',
+  orientation: 'portrait',
+  icon: './src/assets/icon.png',
+  extra: {
+    apiUrl: config.API_URL,
+    apiKey: config.API_KEY,
+    authDomain: config.AUTH_DOMAIN,
+    databaseURL: config.DATABASE_URL,
+    projectId: config.PROJECT_ID,
+    storageBucket: config.STORAGE_BUCKET,
+    messagingSenderId: config.MESSAGING_SENDER_ID,
+    appId: config.APP_ID,
+    clientId: config.CLIENT_ID,
+    androidClientId: config.CLIENT_ID_ANDROID,
+    iosClientId: config.CLIENT_ID_IOS,
+    eas: {
+      projectId: config.EAS_PROJECT_ID,
     },
-    android: {
-      adaptiveIcon: {
-        foregroundImage: './src/assets/adaptive-icon.png',
-        backgroundColor: '#FFFFFF',
-      },
+  },
+  splash: {
+    image: './src/assets/splash.png',
+    resizeMode: 'contain',
+    backgroundColor: '#ffffff',
+  },
+  updates: {
+    fallbackToCacheTimeout: 0,
+    url: `https://u.expo.dev/${config.EAS_PROJECT_ID}`,
+  },
+  assetBundlePatterns: ['**/*'],
+  platforms: ['android', 'ios'],
+  ios: {
+    supportsTablet: true,
+    bundleIdentifier: config.APP_PACKAGE_ID,
+    buildNumber: '3',
+  },
+  android: {
+    versionCode: 3,
```

Comment:
you can bump these versions using the build-command input of the release workflow using https://github.com/nearform/optic-release-automation-action

Reply:
inspire from how we're doing it here https://github.com/nearform/github-snooze-chrome-extension/blob/master/.github/workflows/release.yml